### PR TITLE
Add a method to quickly get a SanitizerBuilder with the default config

### PR DESCRIPTION
**Pull request by Lctrs**
Useful if you want to get all core extensions + your custom extensions

### CODE REVIEW

File: docs/2-creating-an-extension-to-allow-custom-tags.md [hunk 1]
```diff
@@ -135,3 +135,14 @@ $sanitizer = $builder->build([
     'extensions' => ['basic', 'list', 'my-tag'],
 });
 ```
+
+You can also use `HtmlSanitizer\SanitizerBuilder::withDefaultConfig()` to get a `SanitizerBuilder` with all the core extensions:
+
+```php
+$builder = HtmlSanitizer\SanitizerBuilder::withDefaultConfig();
```
Comment (tgalopin) [inline docs/2-creating-an-extension-to-allow-custom-tags.md]: I would prefer to use `createDefault` to be consistent with `Sanitizer::create`, do you mind to update this name :) ?

**Review by tgalopin — COMMENTED** [review]
I like this idea a lot, thanks :) ! Only a small comment.

File: docs/2-creating-an-extension-to-allow-custom-tags.md (hunk 1 above)
Comment (Lctrs) [reply docs/2-creating-an-extension-to-allow-custom-tags.md]: Done.

**Comment by tgalopin** [comment]
Thanks @Lctrs !

**Pull request merged**